### ISSUE TITLE
Templater doesn't run on Daily notes if they are created via sharing

### ISSUE THREAD

**Issue by mnaoumov** [SilentVoid13/Templater#708]
**Plugin information (please complete the following information):**

-   OS: Android 11
-   Templater version: 1.12.0
-   Obsidian version: 1.3.0 (63)
-   Templater settings: default

**Describe the bug**
Templater doesn't run on Daily notes if they are created via sharing.

**Expected behavior**
Templater runs on Daily notes if they are created from the app directly or via sharing

**Screenshots**

https://user-images.githubusercontent.com/735878/181432712-9a97d031-d82f-4502-9113-505cd916490c.mp4

**Additional context**
N/A

**Comment by AB1908** [comment]
Cannot repro on:

```
Obs version 1.3.0 (63)
API: v0.15.6
```

Have you checked Templater's settings on mobile to trigger on new file creation and also set the correct Daily Note template in Daily Note's settings?

**Comment by mnaoumov** [comment]
Yes, I can reproduce this with all the settings mentioned above.

Attached my vault with all settings and plugins for testing.

[vault.zip](https://github.com/SilentVoid13/Templater/files/9227173/vault.zip)

**Comment by shabegom** [comment]
not sure if Templater would be able to hook into that sharing daily note creation. This is an enhancement to existing behaviour.

**Comment by mnaoumov** [comment]
@shabegom it works fine on Windows. It doesn't work on Android

**Comment by TheChilliPL** [comment]
> @shabegom it works fine on Windows. It doesn't work on Android

It doesn't seem to work for me on Windows unless I manually run the Templater command
